domain.go

```diff
@@ -46,13 +50,32 @@ var allowedWWWeTLDs = map[string]bool{
 // To interpret `issues`, see the list of conventions in the
 // documentation for Issues.
 func PreloadableDomain(domain string) (header *string, issues Issues) {
-	header, issues, _ = PreloadableDomainResponse(domain)
+	header, issues, _ = EligibleDomainResponse(domain, Bulk1Year)
+	return header, issues
+}
+
+// EligibleDomain checks whether the domain passes HSTS preload
+// requirements for Chromium when it was add. This includes:
+//
+// - Serving a single HSTS header that passes header requirements.
+//
+// - Using TLS settings that will not cause new problems for
+// Chromium/Chrome users. (Example of a new problem: a missing intermediate certificate
+// will turn an error page from overrideable to non-overridable on
+// some mobile devices.)
+//
+// Iff a single HSTS header was received, `header` contains its value, else
+// `header` is `nil`.
+// To interpret `issues`, see the list of conventions in the
+// documentation for Issues.
+func EligibleDomain(domain string, policy preloadlist.PolicyType) (header *string, issues Issues) {
+	header, issues, _ = EligibleDomainResponse(domain, policy)
 	return header, issues
 }
 
-// PreloadableDomainResponse is like PreloadableDomain, but also returns
+// EligibleDomainResponse is like EligibleDomain, but also returns
 // the initial response over HTTPS.
-func PreloadableDomainResponse(domain string) (header *string, issues Issues, resp *http.Response) {
```

Comment:
This is a change to the public API exposed by this package. I don't see any callsites to it in hstspreload.org, and you've changed all of the callsites in this package, so it's probably fine to make this change.

Since this has a (very slight) risk of breaking backwards compatibility, you should at least put a comment in the commit message mentioning this change. The PR description will become the commit message when the PR gets merged.